.gitignore

```diff
@@ -44,3 +44,7 @@ cairo_venv
 
 # solidity cache
 tmp/
+
+# Setup deps
+setup_and_launch.sh
+./setup_dir
```

Comment:
do we need this in the madara gitignore? looks like specific files for your use case instead of common ignored files